cves/CVE-2010-1157.yml

```diff
@@ -44,45 +44,38 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description: |
-  A portion of Tomcat that handles http authentication via DIGEST and BASIC strategies had the potential to allow
-  for the viewing of the local name or IP address of the server running Tomcat. The header for the authentication 
-  via these strategies includes a <realm-name> element.
-  If a <realm-name> element is specified for the application in one of the configuration files, it will be used. 
-  However, if a <realm-name> is not specified then Tomcat will generate a realm name and
-  in some circumstances this can expose the local host name or IP address of the machine running Tomcat. 
-
+  In an optional header, a user is able to obtain and expose the IP address
```

Comment:
I know you said it was in the optional header but what did the user do with the header? (click, enter a command, ect)

Reply:
Specified the header.